src/main/java/core/basesyntax/service/CsvFormatter.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.TransactionList;
+import core.basesyntax.model.Transaction;
+
+public class CsvFormatter {
+
+    public static void csvToStringArrayList(String csvString) {
```

Comment:
It's better not to use `static`  https://medium.com/att-israel/should-you-avoid-using-static-ae4b58ca1de5 
Lets remove static from all classes in the programm